static/js/filter.js

```diff
@@ -988,4 +999,17 @@ export class Filter {
     static describe(operators) {
         return Handlebars.Utils.escapeExpression(Filter.describe_unescaped(operators));
     }
+
+    static is_web_public_compatible(ops) {
```

Comment:
Does this need to accept a list of operators, or should it just test the operators registered on the `filter` object?

Reply:
Actually, I wanted this function to be independent of Filter, so that I can call this on any narrow without creating a `filter` object.

Reply:
Well, that should be accessible via e.g. `message_list.filter.foo()`?